samples/javascript/web/inventory/app.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express')
+const app = express()
+const apiController = require('./controllers/api')
+const dotenv = require('dotenv')
+const bodyParser = require('body-parser')
+const cookieParser = require('cookie-parser')
+
+dotenv.load({ path: '.env' });
+
+// NOTE: As this is a demo we're referring to files in node_modules which is not something for production purposes.
+app.use('/clarity-ui', express.static(__dirname + '/node_modules/clarity-ui/'));
+
+app.use(cookieParser());
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.set('views', './views');
+app.set('view engine', 'pug');
+
+// Login page (home page)
+app.get('/', function (req, res) {
```

Comment:
I prefer to use the shorthand function syntax. I'll leave it up to you whether you want to change it or not though. It's not a big deal and only syntax.

Example:
```javascript
app.get('/', (req, res) => { ... });
```